<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab 2 - Shader programs</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>621</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="lab2">
     <property name="text" stdset="0">
      <string/>
     </property>
     <attribute name="title">
      <string>Transformations / Vertex Shaders</string>
     </attribute>
     <widget class="QWidget" name="lab2Canvas" native="true">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>341</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="lab3">
     <attribute name="title">
      <string>Fragment Shaders</string>
     </attribute>
     <widget class="QGroupBox" name="shaderProgram">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>141</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Shader Program</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>91</width>
         <height>86</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="solidShader">
          <property name="text">
           <string>Solid</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="gradientShader">
          <property name="text">
           <string>&amp;Gradient</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="textureShader">
          <property name="text">
           <string>Te&amp;xture</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="lab3Canvas" native="true">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>331</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
